src/common/components/ui/formik/validationSchemas.ts

```diff
@@ -21,8 +21,8 @@ import { TFunction } from 'i18next';
 const ALPHANUMBERIC_REGEX = /^[a-zA-Z0-9]+$/;
 const NAME_START_END_REGEX = /^[a-z0-9](.*[a-z0-9])?$/;
 const NAME_CHARS_REGEX = /^[a-z0-9-.]*$/;
-const CLUSTER_NAME_CHARS_REGEX = /^[a-z0-9-]*$/;
-const CLUSTER_NAME_REGEX = /^[a-z0-9](.*[a-z0-9])?$/;
+const CLUSTER_NAME_CHARS_REGEX = /^[a-z0-9-.]*$/;
```

Comment:
Need to tweak this regex further to match https://github.com/openshift/assisted-service/pull/4270/files#diff-72619caf255df33ccaa77f4a5ea4b9980c41cea8fbbbde96f8c62e9085282195R32
(no multiple dots, no combination of hypen and a dot etc.